src/Pagination/utils/ContainerUtils.ts

```diff
@@ -49,9 +51,30 @@ export const hideLoader = (targetListView: DataSourceHelperListView) => {
     targetListView.domNode.classList.remove("widget-pagination-loading");
 };
 
-export const mxTranslation = (namespace: string, key: string, replacements: any[]) => {
-    const templateString = mx.session.getConfig(`uiconfig.translations.${namespace}.${key}`)
-        || (window.mx.session.getConfig("uiconfig.translations") as any)[`${namespace}.${key}`]
-        || "[No translation]";
-    return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
+export const mxTranslation = (namespace: string, key: string, replacements: any[], lookAtWindow: boolean) => {
+    if (!lookAtWindow) {
+        const templateString = mx.session.getConfig(`uiconfig.translations.${namespace}.${key}`)
+            || (window.mx.session.getConfig("uiconfig.translations") as any)[`${namespace}.${key}`]
+            || "[No translation]";
+        return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
+    } else if (window.translations) {
+        const templateString = window.translations[`${namespace}.${key}`]
+            || "[No translation]";
+        return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
+    } else {
+        return "[No translation]";
+    }
+};
+
+export const getTranslations = async (): Promise<void> => {
+    const localeCode = window.mx.session.getConfig("locale.code");
+    const metaData = await fetch("/metamodel.json");
```

Comment:
make sure client does not die, only the widget if the URL is not available

Reply:
Fixed